### ISSUE TITLE
Regarding the 'waifu' system

### ISSUE THREAD

**Issue by Molikai** [Mantaro/MantaroBot#320]
Regarding my background (It's aplpicable to understand my viewpoint) I've been in the BDSM world as a dominant for the better part of the last twenty years. Something we're big on? Consent.
And lo and behold - you're silly little waifu system doesn't care about consent.
And yes, I saw the warning on your help page that says it should not be taken as reality!
A) People don't read it.
B) You think that matters when an abusive asshole is messing with someone's head?
terms like 'gaslighting' are popular, but the correct one is psychological manipulation. 

Might I suggest, as a simple little fix - You either a) Make the supposed 'claim key' work (We tested: It doesn't) as a start: and as a fdinally, either an opt-out (That both blocks and removes all claims) or a simple way for the claimed to say 'no'. 
I've alreay decided that those 'silly little 'commands based dom/sub games that don't bother with consent make me never want to visit your server. 
But perhaps for everyone else, given it's freely available, and you don't know who else is using it?
Or under what circumstances?
I mean, Hyptohyetically, we're talking the grooming of underage girls here, if we want some hyperbole. 
Which might not be.
FIX IT.

**Comment by Kodehawa** [comment]
profile claimlock

**Comment by MrLar** [comment]
> I've alreay decided that those 'silly little 'commands based dom/sub games that don't bother with consent make me never want to visit your server.

Ah yes because the system that clearly states to not take it serious is a "silly command" "based" on dom/sub. And totally not a command that just exist for people to have fun with with 0 implicit. Quite frankly there is no real need for consent, especially given that you can easily opt-out and for that fact that it implies absolutely nothing whatsoever. It only ever gets meaning if you yourself apply meaning to it, at which point you should probably consider the fact that your own viewpoint might be a little wrong here not our system. 

> I mean, Hyptohyetically, we're talking the grooming of underage girls here, if we want some hyperbole.
Which might not be.

You literally called this an exaggeration yourself, if someone ever goes there it is quite frankly just stupid. Sorry for saying that bluntly. But by no means are we implying anything with this system. However anyone who feels disturbed by it can freely disable it or just stop using the bot. Please do note that I am 99% sure that in actual extreme situations (we are talking real harassment and what not here) we would almost certainly take actions and remove them for you and even elevate the issue to discord.

tl;dr This is just blow out of proportion as hell.

**Comment by Molikai** [comment]
The fact you call the possibility 'stupid' just speaks to your innocence. 
Mostly, it suggests your lack of imagination. 
But thank you for answering my question regarding yon claim key!
I'll be sure to take note of this attitude in future.
     On Thursday, 21 May 2020, 20:52:22 BST, Lars <notifications@github.com> wrote:  
 
 



I've alreay decided that those 'silly little 'commands based dom/sub games that don't bother with consent make me never want to visit your server.


Ah yes because the system that clearly states to not take it serious is a "silly command" "based" on dom/sub. And totally not a command that just exist for people to have fun with with 0 implicit. Quite frankly there is no real need for consent, especially given that you can easily opt-out and for that fact that it implies absolutely nothing whatsoever. It only ever gets meaning if you yourself apply meaning to it, at which point you should probably consider the fact that your own viewpoint might be a little wrong here not our system.


I mean, Hyptohyetically, we're talking the grooming of underage girls here, if we want some hyperbole.
Which might not be.


You literally called this an exaggeration yourself, if someone ever goes there it is quite frankly just stupid. Sorry for saying that bluntly. But by no means are we implying anything with this system. However anyone who feels disturbed by it can freely disable it or just stop using the bot. Please do note that I am 99% sure that in actual extreme situations (we are talking real harassment and what not here) we would almost certainly take actions and remove them for you and even elevate the issue to discord.

tl;dr This is just blow out of proportion as hell.

**Comment by Kodehawa** [comment]
"Mostly, it suggests your lack of imagination."

That sounds 20 shades of wrong, lol.

Anyway, there's an opt-out system in place for 5.5.5. I don't like the fact I *had* to add it, but it does exactly what you suggested, makes you unclaimeable and removes all of the claims on you.

**Comment by Kodehawa** [comment]
Also this will mean there's now two ways to opt-out: A claim key applied to your profile with profile claimlock, which only locks you out of new claims and lets you use the waifu system as usual otherwise, and opting out completely, which removes all of the claims on you, but you can't use the system anymore.

Keep in mind that if someone is using the system in bad faith, we can also take action ourselves without you needing to opt-out.

**Comment by Devoxin** [comment]
imagine claiming to be part of the bdsm world for at least 20 years and posting an issue like some snowflake

you spelled submissive wrong btw

**Comment by Molikai** [comment]
That's good to know, thank you.

And that requires you to know someone is using it in bad faith, alas.

Regards,
Douglas
     On Thursday, 21 May 2020, 21:22:09 BST, Kodehawa <notifications@github.com> wrote:  
 
 


Also this will mean there's now two ways to opt-out: A claim key applied to your profile with profile claimlock, which only locks you out of new claims and lets you use the waifu system as usual otherwise, and opting out completely, which removes all of the claims on you, but you can't use the system anymore.

Keep in mind that if someone is using the system in bad faith, we can also take action ourselves without you needing to opt-out.

**Comment by Kodehawa** [comment]
It just requires the person to report it accordingly. If they wanna do it without reporting, they can opt-out as said.
